package.json

```diff
@@ -16,6 +16,7 @@
     "deploy:studio-arbitrum-one": "yarn prepare:arbitrum-one && graph deploy --studio art-blocks-arbitrum",
     "deploy:alchemy-sepolia-dev": "yarn prepare:sepolia-dev && export $(cat .env | xargs) && graph deploy art-blocks-sepolia-dev --node https://subgraphs.alchemy.com/api/subgraphs/deploy --ipfs https://ipfs.satsuma.xyz --deploy-key $ALCHEMY_DEPLOY_KEY",
     "deploy:alchemy-sepolia-staging": "yarn prepare:sepolia-staging && export $(cat .env | xargs) && graph deploy art-blocks-sepolia-staging --node https://subgraphs.alchemy.com/api/subgraphs/deploy --ipfs https://ipfs.satsuma.xyz --deploy-key $ALCHEMY_DEPLOY_KEY",
+    "deploy:chainstack-sepolia-dev": "yarn prepare:sepolia-dev && export $(cat .env | xargs) && graph deploy sgr-918-426-596 --node https://api.graph-ams.p2pify.com/$CHAINSTACK_KEY/deploy --ipfs https://api.graph-ams.p2pify.com/$CHAINSTACK_KEY/ipfs",
```

Comment:
can you add the new env var to env example?

Reply:
thanks! added